### DISCUSSION TITLE
Is it possible for depends_on to define HEAD as the required version?

### DISCUSSION THREAD

**Discussion by wrobelda** [Homebrew/discussions#729] [Everyday usage] [answered]
My project (A) is heavily tied with another one (B), for which we're providing a separate Formula. Both are maintained by us. The HEAD of the main project (A), therefore, would make perfect sense to depend on the HEAD of the project B.  

I tried `depends_on "B" => "HEAD"`, but HEAD is ignored in this case. I couldn't find any other Formula that has a similar use case. Is this supported at all?

**Comment by SMillerDev** [comment, marked as answer]
It's not, head builds aren't really meant for wide usage so we don't have syntax for them beyond defining it per formula.
